doc/nrf/shortcuts.txt

```diff
@@ -243,3 +243,6 @@
 
 .. |ATv2_maintenance_note| replace:: The Asset Tracker v2 application is in maintenance mode.
    For new projects, it is recommended to use the :ref:`nrf_cloud_multi_service` sample instead.
+
+.. |toolchain_management_ncs_versions| replace:: The Toolchain Manager installation is recommended only when using the nRF54H20 DK with the |NCS| v2.7.0.
```

Comment:
Should "v2.7.0" be replaced with "|release|"? Or is that supposed to be left only for v2.7.0?

Reply:
Only for v2.7.0, AFAIK -- info from @FrancescoSer .